### DISCUSSION TITLE
Issue with Redirect after Login - Cal.com Docker Setup

### DISCUSSION THREAD

**Discussion by shwetd19** [calcom/docker#372] [General]
**Description:**
I'm experiencing an issue with my Cal.com instance hosted on a GCP VM. After logging in, the application redirects to `http://api/auth/error` and displays the following message: "Something went wrong. Please try again and contact us if the issue persists."

Here are the steps I've followed for the installation and setup:

**Installation Steps:**

1. **Installation of Docker and Docker Compose via a Simple Script**
   - Download the installation script:
     ```bash
     wget -O install-docker.sh https://gitlab.com/bmcgonag/docker_installs/-/raw/main/install_docker_nproxyman.sh
     ```
   - Change the permissions to make the script executable:
     ```bash
     chmod +x ./install-docker.sh
     ```
   - Run the script:
     ```bash
     ./install-docker.sh
     ```
   - Follow the prompts to install Docker-CE and Docker Compose. Optionally, install NGinX Proxy Manager.

2. **Installing Cal.com**
   - Clone the Cal.com Docker repository:
     ```bash
     git clone https://github.com/calcom/docker.git
     ```
   - Rename the folder for clarity:
     ```bash
     mv docker cal-com-docker
     ```
   - Navigate to the project directory:
     ```bash
     cd cal-com-docker
     ```
   - Copy the example environment file and edit it:
     ```bash
     cp .env.example .env
     nano .env
     ```
   - Set the environment variables as needed. Here are some key variables I configured:
     ```plaintext
     NEXT_PUBLIC_LICENSE_CONSENT=true
     NEXT_PUBLIC_WEBAPP_URL=https://cal.swades.ai
     ```
   - Edit the `docker-compose.yaml` file to change the port mapping:
     ```plaintext
     services:
       calcom:
         ports:
           - "8594:3000"
     ```
   - Update the Cal.com Submodule:
     ```bash
     git submodule update --remote --init
     ```
   - Build and Start Cal.com via Docker Compose:
     ```bash
     docker compose up --build
     ```  

3. **Setup our Reverse Proxy**

Despite following these steps, I'm encountering the redirect issue after login.

**Additional Information:**
- Here is a [Loom recording](https://www.loom.com/share/6c46f0ee78134d65bb16f9d30c7968c7?sid=22d889bc-249e-485c-b564-4c10ca33516d) showing the issue and the steps I've taken.

**Help Needed:**
I would appreciate any insights or suggestions on what might be causing this issue and how to resolve it. Thank you in advance!

**Comment by rimallo** [comment]
Exactly the same issue. Calcom is a great solution which never worked on my own subdomains without a port behind reverse proxy.

**Comment by rimallo** [comment]
Actually current docker repo works `somehow` only in build variant (for advanced user). Out-of-the-box solution hangs on creating DB.
